lms/djangoapps/verify_student/models.py

```diff
@@ -1108,3 +1109,23 @@ def deadline_for_course(cls, course_key):
             return deadline.deadline
         except cls.DoesNotExist:
             return None
+
+
+class SSOReverificationConfig(ConfigurationModel):
```

Comment:
I believe this is SSPV not SSO, rename it to `SSReverificationConfig ` here and everywhere else.